SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -74,6 +75,16 @@ struct FSubobjectSchemaData
 	}
 };
 
+// Schema data related to a schema component
+USTRUCT()
+struct FComponentSchemaData
+{
+	GENERATED_USTRUCT_BODY()
+
+	UPROPERTY(Category = "SpatialGDK", VisibleAnywhere)
+	TArray<uint32> FieldIds;
```

Comment:
uint8?

Reply:
You can have a lot of fields on one component.

Reply:
I tried to keep the same sizes as defined by the WorkerSDK. Also wondering if the compiler will warn us if these change, and if we maybe WANT to deviate.

Reply:
If worried about the worker sdk changing them use a FieldId type.

Reply:
Ah, I think this is a funny thing. Even though both uint32 and Schema_FieldId resolve to uint32_t, Unreal can't handle Schema_FieldId in a serializable UPROPERTY for some reason. We have had headaches with this in the past.

Reply:
We might want to just define global types of SpatialGDK::FieldId and EntityId that use the unreal widths to save us from this.